examples/3d/debug_gltf.rs

```diff
@@ -0,0 +1,106 @@
+use bevy::{
+    gltf::{Gltf, GltfNode},
+    prelude::*,
+};
+
+fn main() {
+    App::new()
+        .insert_resource(AmbientLight {
+            color: Color::WHITE,
+            brightness: 1.0 / 5.0f32,
+        })
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup)
+        .add_system(spawn_and_print_gltf_objects)
+        .add_system(assert_correctness)
+        .run();
+}
+
+fn setup(mut commands: Commands, assets: Res<AssetServer>) {
+    // Gltf can be packed in a more compact format called glb.
+    // Unlike Gltf this format isn't human readable
+    // so being able to print the loaded structure is extra valuable
+    let handle: Handle<Gltf> = assets.load("models/AlienCake/alien.glb");
+    commands.insert_resource(handle);
+    commands.insert_resource((false, false));
+
+    commands.spawn_bundle(PerspectiveCameraBundle {
+        transform: Transform::from_xyz(0.5, 1.0, -2.0)
+            .looking_at(Vec3::new(0.0, 0.3, 0.0), Vec3::Y),
+        ..Default::default()
+    });
+    const HALF_SIZE: f32 = 1.0;
+    commands.spawn_bundle(DirectionalLightBundle {
+        directional_light: DirectionalLight {
+            shadow_projection: OrthographicProjection {
+                left: -HALF_SIZE,
+                right: HALF_SIZE,
+                bottom: -HALF_SIZE,
+                top: HALF_SIZE,
+                near: -10.0 * HALF_SIZE,
+                far: 10.0 * HALF_SIZE,
+                ..Default::default()
+            },
+            shadows_enabled: true,
+            ..Default::default()
+        },
+        transform: Transform::from_xyz(-0.7, 1.7, -2.0)
+            .looking_at(Vec3::new(0.0, 0.3, 0.0), Vec3::Y),
+        ..Default::default()
+    });
+}
+
+pub fn spawn_and_print_gltf_objects(
+    mut commands: Commands,
+    mut done: ResMut<(bool, bool)>,
```

Comment:
Two things:
1. This is probably going to panic, as the resource is never inserted.
2. We should try to use wrapper structs, rather than raw data types like this, as it makes intent much clearer.

Reply:
1. The resource got inserted in [setup](https://github.com/bevyengine/bevy/blob/4f3afc2bf727d1d2e70aabe12dae7e4ed3699320/examples/3d/debug_gltf.rs#L25), which wasn't idiomatic.
2. Done :)